### PR TITLE
Adding a way to see Security Group Rules

### PR DESCRIPTION
**Pull request by SrinivasChilveri**
Before Change :  Users were not able to see Security Group Rules applied to their spaces
After Change : Users(Admin and Non-admin) will be able to see the rules associated with the security groups applied to their spaces.

$ ./cf space mySpace --security-group-rules
Getting info for space mySpace in org huawei as asif...
OK

mySpace
       Org:               huawei
       Apps:
       Domains:           10.244.0.34.xip.io
       Services:
       Security Groups:   dns
       Space Quota:

Getting rules for the security group  : dns...

   destination   :   0.0.0.0/0
   ports         :   53
   protocol      :   tcp

   destination   :   0.0.0.0/0
   ports         :   53
   protocol      :   udp

### CODE REVIEW

**Comment by asifhuawei** [comment]
This pull request solves the problem reported in https://www.pivotaltracker.com/n/projects/892938/stories/77733680

It was suggested to add this flag in either 'app' command or 'space' command but we choosed space command beacuse it was already having this information so we just need to format the data in human readable format and present it, where as in app command we were not having the security group rules information so we need to go one more round to CC to get these data which would be an overhead so we decide to go with the "space mySpace --security-group-rules".
And this flag will be accessible by both admin & non-admin users.

**Comment by goehmen** [comment]
Hi Srinivas.  I move the story to the backlog & it's fairly high in priority.  I had a similar story about IP table visibilities and want to verify that your PR fulfills the requirements for that as well.

**Comment by asifhuawei** [comment]
Hi @goehmen 
The present changes in this PR will show all the information of IPTables in the json format structure but if you want the data in the tabular format as illustrated in the IPTables story(#76616638)  then we will be happy to add the same feature for 'cf security-groups' and we will present the data there in tabular format.
Waiting for your approval to make changes in this PR......

**Comment by goehmen** [comment]
The tacker story is now unblocked & we are good to go

File: cf/commands/space/space_test.go [hunk 1]
```diff
@@ -112,6 +112,16 @@ var _ = Describe("space command", func() {
 			})
 		})
 
+		Context("when the security-group-rules flag is passed", func() {
+			It("it shows space information and security group rules", func() {
+				runCommand("--security-group-rules", "whose-space-is-it-anyway")
+
+				Expect(ui.Outputs).To(ContainSubstrings(
+					[]string{"Getting rules for the security group"},
```
Comment (simonleung8) [inline cf/commands/space/space_test.go]: Looks like the test only cover the title of the table. We would like the test to also assert the content of the table is correctly printed. Do you mind updating the test a little?

**Comment by asifhuawei** [comment]
@simonleung8  Thanks for the review comment......we will be happy to make changes to this PR

**Comment by asifhuawei** [comment]
@simonleung8  We have made the test case assertion more specific to the output, please review the changes

**Comment by simonleung8** [comment]
@asifhuawei Thanks for updating the test, merged.

**Pull request merged**